src/lib.rs

```diff
@@ -243,6 +257,7 @@ impl<'object, Endian> DebugInfo<'object, Endian>
                 }
 
                 // This program covers the given address -- calculate how well it matches
+                // NOTE: the unwrap is okay, because p.contains_address returned true above.
```

Comment:
Rather than commenting `unwrap()` calls, I prefer to pass what would have been a comment into an `expect("...")` call. That way, if a panic does happen, we have more of a clue what is going on.